### PR TITLE
Add a chapter about available reading material

### PR DESCRIPTION
**Pull request by Jaanki**
Add a new chapter listing all conference talks,
demo recordings and blogs.

Signed-Off-By: Janki Chhatbar <jchhatba@redhat.com>
Closes: #22

### CODE REVIEW

File: src/content/reading_material/_index.en.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+## Demo recordings
+1. [Deploying Submariner with subctl](https://www.youtube.com/watch?v=cInmBXuZsU8&t=17s)
+2. [Connecting hybrid Kubernetes clusters using Submariner](https://www.youtube.com/watch?v=fMhZRNn0fxQ)
+3. [Cross-cluster service discovery in Submariner using Lighthouse](https://www.youtube.com/watch?v=tXsemQPNhyQ)
+
+## Blogs
+1. [Kubernetes Multi-Cloud and Multi-Cluster Connectivity with Submariner](https://www.linkedin.com/pulse/kubernetes-multi-cloud-multi-cluster-connectivity-gokul-chandra/?trk=related_artice_Kubernetes%20Multi-Cloud%20and%20Multi-Cluster%20Connectivity%20with%20Submariner_article-card_title)
+
+If you find any more presentations that are now listed here, please feel free to push a PR adding those videos.
```
Comment (tpantelis) [inline src/content/reading_material/_index.en.md]:
```suggestion
If you find additional material that isn't listed here, please feel free to add it to this page.
```

Comment (mangelajo) [reply src/content/reading_material/_index.en.md]: :+1:

**Comment by mangelajo** [comment]
Nice work @Jaanki you seem to have a conflict on "contributing" and also see tpantelis suggestion :)

File: src/content/reading_material/_index.en.md (hunk 1 above)
Comment (Jaanki) [reply src/content/reading_material/_index.en.md]: Done

**Pull request merged**